test/dummy/config/environments/test.rb

```diff
@@ -52,6 +52,7 @@
   config.solid_queue.connects_to = { database: { writing: :queue } }
 
   config.mission_control.jobs.http_basic_auth_enabled = false
+  config.mission_control.jobs.base_controller_class = 'MyApplicationController'
```

Comment:
```suggestion
  config.mission_control.jobs.base_controller_class = "MyApplicationController"
```
just for consistency with the rest of the gem.